google/resource_compute_instance_group_manager_test.go

```diff
@@ -20,7 +19,7 @@ import (
 func TestAccInstanceGroupManager_basic(t *testing.T) {
 	t.Parallel()
 
-	var manager compute.InstanceGroupManager
+	var manager computeBeta.InstanceGroupManager
```

Comment:
since you're doing test changes anyway, I think it would be worth simplifying the tests by getting rid of the `Exists` function and whichever checks duplicate functionality from import.

Reply:
👍